src/mon/MgrMap.h

```diff
@@ -179,6 +179,29 @@ class MgrMap
     return false;
   }
 
+  bool module_enabled(const std::string &module_name) const
+  {
+    for (const auto &name : modules) {
+      if (name == module_name) {
+        return true;
+      }
+    }
+
+    return false;
+  }
+
+  bool module_existed(const std::string& module) {
```

Comment:
I suggest naming this function `any_support_module` so that its relationship with `all_support_module` is more obvious

Reply:
@jcsp fixed.